CHANGELOG.md

```diff
@@ -0,0 +1,17 @@
+# Changelog
+
+## v1.0.0-beta.0
+
+FEATURES
+[\#5](https://github.com/binance-chain/bsc/pull/5) enable bep2e tokens for faucet
```

Comment:
> [\#5](https://github.com/binance-chain/bsc/pull/5) enable bep2e tokens for faucet

* [\#5](https://github.com/binance-chain/bsc/pull/5) enable bep2e tokens for faucet